### ISSUE TITLE
Unable to determine neovim windows id for editor viewColumn: 1

### ISSUE THREAD

**Issue by David-Else** [asvetliakov/vscode-neovim#418]
I occasionally get an error like this when first trying to open a file:
```
2020-10-28T15:20:07.854Z BufferManager: Unable to determine neovim windows id for editor viewColumn: 1, docUri: file:///home/xxx/xxx/file.ts
```
It does not seem repeatable but might point to a problem.

**Comment by ddxgz** [comment]
Having the same issue, and it shows everytime vscode start up:
```
2020-11-11T09:46:04.567Z BufferManager: Unable to determine neovim windows id for editor viewColumn: undefined, docUri: output:extension-output-%234
```

**Comment by Nico-Guyon** [comment]
I have also the same issue.
It happens sometimes when I open diff view from the scm panel.
This is quite random but it does happens quite a lot.
Closing and reopening the file / view is enough in general to get control back.

`2020-11-11T15:25:53.581Z BufferManager: Unable to determine neovim windows id for editor viewColumn: undefined, docUri: filePath`

I seems to happens to more the faster I change files / views.

**Comment by Nico-Guyon** [comment]
Ok since this is rather annoying might as well try to solve this or at least find some clues !

The problem comes from `src/buffer_manager.ts`

Sometimes `syncActiveEditor` is called but just before `syncLayout` actually deleted the `TextEditor` element from the map `editorColumnsToWinId`.
`syncLayout` is called after and set back the correct `TextEditor` value inside the map.

I'll investigate more when I have more time, but it looks like some kind of side effect bug.

**Comment by asvetliakov** [comment]
@Nico-Guyon thank you for the investigation. This error is harmless now, so i just downgrade report level

Released 0.0.71 with the fix
